README.md

```diff
@@ -1,4 +1,4 @@
-# Effect Foundation API
+## Effect Foundation API
```

Comment:
Switched to ## so my auto-table-of-contents plugin can include this, and doesn't indent everything else